example-app/src/main/java/com/adyen/checkout/example/data/api/model/PaymentsRequest.kt

```diff
@@ -30,13 +30,12 @@ data class PaymentsRequestData(
     val countryCode: String,
     val merchantAccount: String,
     val returnUrl: String,
-    val additionalData: AdditionalData,
+    val authenticationData: AuthenticationData,
     val threeDSAuthenticationOnly: Boolean,
     val shopperIP: String,
     val reference: String,
     val channel: String,
     val lineItems: List<Item>,
     val shopperEmail: String? = null,
-    val threeDS2RequestData: ThreeDS2RequestDataRequest?,
```

Comment:
I see that [this field](https://docs.adyen.com/api-explorer/Checkout/71/post/payments#request-threeDS2RequestData) is not deprecated here. Since it is replaced with `authenticationData.threeDSRequestData`, what do you think of asking backend to deprecated it?
Though they do not look like the same objects to me.

Reply:
The object is not deprecated but apparently not applicable for checkout, so we should not be using it.